client/network/src/peer_info.rs

```diff
@@ -186,6 +186,17 @@ impl NetworkBehaviour for PeerInfoBehaviour {
 		list
 	}
 
+	fn inject_address_change(
+		&mut self,
+		_: &PeerId,
+		_: &ConnectionId,
+		_: &ConnectedPoint,
+		_: &ConnectedPoint,
+	) {
+		// NetworkBehaviour::inject_address_change on Ping does nothing.
+		// NetworkBehaviour::inject_address_change on Identify does nothing.
+	}
```

Comment:
Please do it anyway. The fact that Ping and Identify do nothing is an implementation detail.